ceph-releases/jewel/daemon/check_zombie_mons.py

```diff
@@ -23,25 +30,30 @@ def extract_mons_from_monmap():
             mons[m.group(2)] = m.group(1)
     return mons
 
+
 def extract_mons_from_kubeapi():
     kubemap = subprocess.check_output(kubectl_command, shell=True)
     return json.loads(kubemap)
 
+
 current_mons = extract_mons_from_monmap()
 expected_mons = extract_mons_from_kubeapi()
 
-print "current mons:", current_mons
-print "expected mons:", expected_mons
+print 'current mons:{}'.format(current_mons)  # noqa E999
+print 'expected mons:{}'.format(expected_mons)  # noqa E999
 
 for mon in current_mons:
     removed_mon = False
-    if not mon in expected_mons:
+    if mon not in expected_mons:
         print "removing zombie mon ", mon
-        subprocess.call(["ceph", "--cluster", os.environ["CLUSTER"], "mon", "remove", mon])
+        subprocess.call(["ceph", "--cluster", os.environ["CLUSTER"], "mon",
+                        "remove", mon])
         removed_mon = True
-    elif current_mons[mon] != expected_mons[mon]: # check if for some reason the ip of the mon changed
+        # check if for some reason the ip of the mon changed
```

Comment:
Should be left-aligned with the `elif`. It could be unclear the comment applies to the `elif` and not the content above.